config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -298,6 +298,6 @@ periodics:
       securityContext:
         privileged: true
   annotations:
-    testgrid-dashboards: sig-windows
+    testgrid-dashboards: sig-windows, provider-azure-on-call
```

Comment:
can you skip the staging job? that's used to test new things before they're added to the above jobs.

Reply:
Can you pls also remove the testgrid email alert for the staging job? Thx!

Reply:
sure!

Reply:
Removed the testgrid email alert for staging job.

@ritazh PTAL. Thanks!